CCCS_YARA.yml

```diff
@@ -206,6 +208,8 @@ mitre_att:
    unique: No
    optional: Yes
    validator: valid_mitre_att
+   argument:
+      softCodesFound:
```

Comment:
whats the meaning of softcodes?

Reply:
perhaps mitre_softwareid would be a better name

Reply:
I will expand on the description as well. Those are being left there to indicate that these are fields that will be utilized by the validator to store found mitre_att software ids.

Reply:
Per the commit below I changed some of the names for more clarity:
  ifKeyGenMitreAtt -> if_key_gen_mitre_att
  malwareIdsFound  -> mitre_softwareid_gen
  softCodesFound   -> mitre_softwareid_found
I think those are a little clearer, I also added the support for tool: "toolname" metadata tags to auto generate mitre_att software codes if not present.